GenumKit/Stencil/Filters.swift

```diff
@@ -80,12 +80,32 @@ struct StringFilters {
 
 struct ArrayFilters {
   static func join(value: Any?) throws -> Any? {
-    guard let array = value as? [Any] else { throw FilterError.InvalidInputType }
-    let strings = array.flatMap { $0 as? String }
-    guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    let strings: [String]
+    if let array = value as? [Any] {
+      strings = array.flatMap { $0 as? String }
+      guard array.count == strings.count else { throw FilterError.InvalidInputType }
+    } else if let array = value as? [String] {
+      strings = array
+    } else {
+      throw FilterError.InvalidInputType
+    }
 
     return strings.joinWithSeparator(", ")
   }
+
+  static func joinAnd(value: Any?) throws -> Any? {
```

Comment:
@kylef sadly, I can't upgrade stencil because SwiftGen inherits some classes to modify some behavior. Since things aren't open, this won't compile.

Take a look at https://github.com/AliSoftware/SwiftGen/blob/master/GenumKit/Stencil/GenumTemplate.swift

Template and Namespace are both subclassed here. The code seems reasonable to me (so wondering if you'd consider making these open), but wondering if you have other ideas.